src/test/java/com/plaid/client/integration/depositswitch/DepositSwitchCreateTest.java

```diff
@@ -0,0 +1,49 @@
+package com.plaid.client.integration.depositswitch;
+
+import com.plaid.client.integration.AbstractIntegrationTest;
+import com.plaid.client.request.AccountsGetRequest;
+import com.plaid.client.request.ItemImportRequest;
+import com.plaid.client.request.depositswitch.DepositSwitchCreateRequest;
+import com.plaid.client.response.AccountsGetResponse;
+import com.plaid.client.response.ItemImportResponse;
+import com.plaid.client.response.depositswitch.DepositSwitchCreateResponse;
+
+import org.junit.Test;
+import retrofit2.Response;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+
+public class DepositSwitchCreateTest extends AbstractIntegrationTest {
+  @Test
+  public void testDepositSwitchCreateSuccess() throws Exception {
+    HashMap<String, String> userAuth = new HashMap<>();
+    userAuth.put("user_id", "user_good");
+    userAuth.put("auth_token", "pass_good");
+
+    Response<ItemImportResponse> itemImportResponse = client().service()
+        .itemImport(new ItemImportRequest(Arrays.asList("identity", "auth"), userAuth)
+            .withWebhook("https://some.webhook.example.com"))
+        .execute();
+
+    assertSuccessResponse(itemImportResponse);
+    assertNotNull(itemImportResponse.body().getAccessToken());
+
+    String accessToken = itemImportResponse.body().getAccessToken();
+
+    // get account id
+    Response<AccountsGetResponse> accountsGetResponse = client().service()
+        .accountsGet(new AccountsGetRequest(accessToken)).execute();
+    assertSuccessResponse(accountsGetResponse);
+    assertNotNull(accountsGetResponse.body().getAccounts());
+
+    String accountId = accountsGetResponse.body().getAccounts().get(0).getAccountId();
```

Comment:
Instead of grabbing the zero'th index, could you filter for an appropriate account type or subtype instead?

Reply:
@rayraycano  is there any reason why we want to filter by a specific account type or subtype instead of getting an arbitrary account? Taking the 0th account is consistent with other tests, like in `TransactionsGetTest.java`

Reply:
For deposit switch, it just so happens that the 0th account returned by Plaid's /accounts/get endpoint is a depository type account. What we actually care about is getting an account of type `depository`, so I would opt to filter for the first account of such type.